--- arch/makefile.include.linux_intel.orig	2020-05-25 18:04:34.000000000 -0700
+++ arch/makefile.include.linux_intel	2020-05-25 19:27:29.000000000 -0700
@@ -11,8 +11,8 @@
 
 CPP        = fpp -f_com=no -free -w0  $*$(FUFFIX) $*$(SUFFIX) $(CPP_OPTIONS)
 
-FC         = mpiifort
-FCL        = mpiifort -mkl=sequential -lstdc++
+FC         = mpifort
+FCL        = mpifort -mkl=sequential -lstdc++
 
 FREE       = -free -names lowercase
 
