See https://github.com/mpimd-csc/flexiblas/issues/102
--- flexiblas-3.5.0/src/xerbla.c.orig	2026-09-14 13:42:28.886874615 +0000
+++ flexiblas-3.5.0/src/xerbla.c	2026-09-14 13:46:45.847929952 +0000
@@ -42,10 +42,8 @@
 #ifdef __ELF__
 #pragma weak xerbla_
 #pragma weak xerbla
-#pragma weak XERBLA
 void xerbla_(char *, Int *, flexiblas_fortran_charlen_t) __attribute__ (( alias ("flexiblas_internal_xerbla")));
 void xerbla(char *, Int *, flexiblas_fortran_charlen_t) __attribute__ (( alias ("flexiblas_internal_xerbla")));
-void XERBLA(char *, Int *, flexiblas_fortran_charlen_t) __attribute__ (( alias ("flexiblas_internal_xerbla")));
 #else
 void xerbla_(char *SNAME, Int *Info, flexiblas_fortran_charlen_t) {
     flexiblas_internal_xerbla(SNAME, Info, len);
@@ -53,9 +51,6 @@
 void xerbla(char *SNAME, Int *Info, flexiblas_fortran_charlen_t len) {
     flexiblas_internal_xerbla(SNAME, Info, len);
 }
-void XERBLA(char *SNAME, Int *Info, flexiblas_fortran_charlen_t len) {
-    flexiblas_internal_xerbla(SNAME, Info, len);
-}
 #endif
 
 #else
@@ -63,7 +58,6 @@
 #if defined(__ELF__) || ((defined (__PGI) || defined(__NVCOMPILER)) && (defined(__linux__)  || defined(__unix__)))
 void xerbla_(char *, Int *, flexiblas_fortran_charlen_t) __attribute__ ((weak, alias ("flexiblas_internal_xerbla")));
 void xerbla (char *, Int *, flexiblas_fortran_charlen_t) __attribute__ ((weak, alias ("flexiblas_internal_xerbla")));
-void XERBLA (char *, Int *, flexiblas_fortran_charlen_t) __attribute__ ((weak, alias ("flexiblas_internal_xerbla")));
 
 #else
 /* #pragma weak xerbla_
