From 693b5fa7b5e671e448431238d12265ac50953551 Mon Sep 17 00:00:00 2001
From: rakhmets <rakhmetshin@nvidia.com>
Date: Fri, 30 May 2025 13:13:40 +0300
Subject: [PATCH] UCT/CUDA/CUDA_COPY: Removed non-versioned typedef.

(backported by Bart Oldeman <bart.oldeman@calculquebec.ca>)

--- ucx-1.14.1/src/uct/cuda/base/cuda_md.c.orig	2026-04-02 12:11:49.732944486 +0000
+++ ucx-1.14.1/src/uct/cuda/base/cuda_md.c	2026-04-02 12:12:06.391911700 +0000
@@ -223,7 +223,7 @@
 static int uct_cuda_copy_get_dmabuf_fd(const ucs_memory_info_t *addr_mem_info)
 {
 #if CUDA_VERSION >= 11070
-    PFN_cuMemGetHandleForAddressRange get_handle_func;
+    PFN_cuMemGetHandleForAddressRange_v11070 get_handle_func;
     const char *cu_err_str;
     CUresult cu_err;
     int fd;
